CCVA_Report_Formatting.Rmd

```diff
@@ -19,27 +19,32 @@ editor_options:
     wrap: 79
 ---
 
-```{r setup, include=FALSE, eval = FALSE}
+```{r setup, include=FALSE}
 # set chunk defaults
 knitr::opts_chunk$set(echo = FALSE,
                       warning = FALSE,
                       error = FALSE,
-                      message = FALSE)
+                      message = FALSE,
+                      fig.width = 7)
                       
                       
 # read in libraries
 source("setup.R")
-
+library(scales)
```

Comment:
Is this our new approach for tackling setup merge conflicts?

Reply:
I've been doing this also but more as a temporary fix -- we can do one bulk add to reduce the number of merge conflicts.

Reply:
First, ignore anything in this formatting .Rmd it is mainly just for me to experiment getting the templates to work. But second, I was following your lead @KECognac and tacking on new libraries into the Report .Rmd under the `source('setup.R')`, but I support doing a bulk add of new libraries to the setup script in the next PR!